### PR TITLE
[BACKLOG-35131] Add HDI40 shim configurations

### PR DESCRIPTION
**Pull request by renjith-nem**

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 19s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
org.apache.maven.model.building.ModelBuildingException: 1 problem was encountered while building the effective model
[FATAL] Non-parseable POM /build/wingman/workspace/maintenance/wingman/builds/pentaho.big-data-plugin.PR-2185/assemblies/legacy-plugin/pom.xml: end tag name </dependency> must match start tag name <dependencies> from line 31 (position: TEXT seen ...</dependency>\n      </dependency>... @39:20)  @ line 39, column 20
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 12m 5s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **1298**, Failures: **0**, Skipped: **2** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-plugin/job/PR-2185/2/testReport)

 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -37,6 +37,7 @@
     <pentaho-hadoop-shims-emr521.version>9.3.0.0-SNAPSHOT</pentaho-hadoop-shims-emr521.version>
     <pentaho-hadoop-shims-dataproc1421.version>9.3.0.0-SNAPSHOT</pentaho-hadoop-shims-dataproc1421.version>
     <pentaho-hadoop-shims-cdpdc71.version>9.3.0.0-SNAPSHOT</pentaho-hadoop-shims-cdpdc71.version>
+    <pentaho-hadoop-shims-hdi40.version>9.2.0.0-SNAPSHOT</pentaho-hadoop-shims-hdi40.version>
```
Comment (ddiroma) [inline pom.xml:40]: @renjith-nem this should be 9.3.0.0-SNAPSHOT

**Comment by buildguy** [comment]
### :x: Build failed in 13m 45s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **1298**, Failures: **0**, Skipped: **2** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-plugin/job/PR-2185/3/testReport)

 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Review by ddiroma — APPROVED** [review]

**Pull request merged**